client/layout/store-alerts/index.js

```diff
@@ -0,0 +1,179 @@
+			url: '#',
+		},
+	},
+];
+
+const alertTypes = {
+	emergency: {
+		label: __( 'Emergency', 'wc-admin' ),
+		icon: 'warning',
+	},
+	alert: {
+		label: __( 'Alert', 'wc-admin' ),
+		icon: 'flag',
+	},
+	critical: {
+		label: __( 'Critical', 'wc-admin' ),
+		icon: 'info',
+	},
+};
+
+class StoreAlerts extends Component {
+	constructor( props ) {
+		super( props );
+		const { alerts } = this.props;
+
+		this.state = {
+			currentIndex: alerts ? 0 : null,
+		};
+
+		this.previousAlert = this.previousAlert.bind( this );
+		this.nextAlert = this.nextAlert.bind( this );
+	}
+
+	previousAlert( event ) {
+		event.stopPropagation();
+		const { currentIndex } = this.state;
+
+		if ( currentIndex > 0 ) {
+			this.setState( {
+				currentIndex: currentIndex - 1,
+			} );
+		}
+	}
+
+	nextAlert( event ) {
+		event.stopPropagation();
+		const { alerts } = this.props;
+		const { currentIndex } = this.state;
+
+		if ( currentIndex < alerts.length - 1 ) {
+			this.setState( {
+				currentIndex: currentIndex + 1,
+			} );
+		}
+	}
+
+	render() {
+		const { alerts } = this.props;
+		const { currentIndex } = this.state;
+		const numberOfAlerts = alerts.length;
+		const alert = alerts[ currentIndex ] ? alerts[ currentIndex ] : null;
+		const type = alert && alert.type ? alert.type : null;
+		const icon = type && alertTypes[ type ] ? alertTypes[ type ].icon : null;
+		const className = classnames( 'woocommerce-store-alerts', {
+			'is-alert-emergency': 'emergency' === type,
+			'is-alert-alert': 'alert' === type,
+			'is-alert-critical': 'critical' === type,
+		} );
+
+		return (
+			numberOfAlerts > 0 && (
+				<Card
+					title={ [
+						icon && <Dashicon key="icon" icon={ icon } />,
+						<Fragment key="title">{ alert.title }</Fragment>,
+					] }
+					className={ className }
+					action={
+						numberOfAlerts > 1 && (
+							<div className="woocommerce-store-alerts__pagination">
+								<IconButton
+									icon="arrow-left-alt2"
+									onClick={ this.previousAlert }
+									disabled={ 0 === currentIndex }
+									label={ __( 'Previous Alert', 'wc-admin' ) }
+								/>
+								<span
+									className="woocommerce-store-alerts__pagination-label"
+									role="status"
+									aria-live="polite"
+								>
+									{ interpolateComponents( {
+										mixedString: __( '{{current /}} of {{total /}}', 'wc-admin' ),
+										components: {
+											current: <Fragment>{ currentIndex + 1 }</Fragment>,
+											total: <Fragment>{ numberOfAlerts }</Fragment>,
+										},
+									} ) }
+								</span>
+								<IconButton
+									icon="arrow-right-alt2"
+									onClick={ this.nextAlert }
+									disabled={ numberOfAlerts - 1 === currentIndex }
+									label={ __( 'Next Alert', 'wc-admin' ) }
+								/>
+							</div>
+						)
+					}
+				>
+					<div className="woocommerce-store-alerts__message">{ alert.message }</div>
+					<Button isPrimary className="woocommerce-store-alerts__button" href={ alert.action.url }>
```

Comment:
Can you change this to `isDefault` ? 

There's some funky override going on in there somewhere, when I change it to `isDefault` the normal state button text is white, when it should be $core-grey-dark-500`

Reply:
I'm also seeing the same issues on mobile that are noted above.

I'd like the left margin to match the rest of the content (it should be 16px on the smallest screens) and position the segmented button above the alert title? 

<img width="648" alt="screen shot 2019-02-21 at 2 23 44 pm" src="https://user-images.githubusercontent.com/4500952/53206071-df7f7a80-35e4-11e9-807e-34fb400ae4ff.png">